version-select.js

```diff
@@ -0,0 +1,164 @@
+window.addEventListener("DOMContentLoaded", function() {
+    window.versionPages = {};
+    var VERSION = window.location.pathname.split("/")[1];
+    var VERSION_LATEST = '.';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/codacy/docs/pullRequest?prid=26920)
```suggestion
b2d185ebcfa2779386bed1f64af3965ceff3c4b1f05ffe513185298517713b28
```